admin/c2cgeoportal_admin/routes.py

```diff
@@ -39,18 +39,33 @@ def includeme(config):
     )
     from c2cgeoportal_commons.models.static import User
 
+    default_routes = (
+        ('themes', Theme),
+        ('layer_groups', LayerGroup),
+        ('layers_wms', LayerWMS),
+        ('layers_wmts', LayerWMTS),
+        ('ogc_servers', OGCServer),
+        ('restriction_areas', RestrictionArea),
+        ('users', User),
+        ('roles', Role),
+        ('functionalities', Functionality),
+        ('interfaces', Interface),
+    )
+    visible_routes = default_routes
+
+    if 'include_pages' in config.registry.settings['admin_interface']:
+        include_pages = config.registry.settings['admin_interface']['include_pages']
+        for route_name, class_path in include_pages:
```

Comment:
Note this is not exactly a route_name, but a value for the table parameter in all c2cgeoform routes.
So `url_path` may be more precise.